kalite/control_panel/views.py

```diff
@@ -539,6 +537,7 @@ def _get_user_usage_data(users, groups=None, period_start=None, period_end=None,
 
     # Add group data.  Allow a fake group UNGROUPED
     for user in users:
+        user_data[user.pk]["pct_mastery"] = user_data[user.pk]["pct_mastery"]/(user_data[user.pk]["total_exercises"] or 1)
```

Comment:
Previously % mastery was being calculated for mastery of *every single exercise*, seemed like a high bar.